lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -85,9 +85,9 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
 
   defp handle_req("metadata/2/ssz_snappy", message_id, _message) do
     # Values are hardcoded
-    with {:ok, payload} <-
-           <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>
-           |> Snappy.compress() do
+    with metadata <- P2P.Metadata.get_metadata(),
+         {:ok, payload} <- Ssz.to_ssz(metadata),
+         {:ok, payload} <- Snappy.compress(payload) do
```

Comment:
Let's not repeat variable names, it's very error prone. We can have "serialized" for the first one and "payload" for the second one.